### ISSUE TITLE
Замечания 1

### ISSUE THREAD

**Issue by Sherevv** [mrognor/Praktika1#1]
Почему print сделан дружественной функцией, а не методом класса?
Конструкторы дочернего класса можно упростить, вызавая конструкторы базового класса:
Drob_Child(int Chisl): Drob(Chisl){}
Если передаете параметры по ссылке и они не должны меняться (в большинстве случаев это так), то лучше передавать по константной ссылке
const Drob& a

Пишите в коде вариант и текст задания. И вообще комментарии будут не лишними.
И оставьте только файлы кода, без проекта.

**Comment by mrognor** [comment]
Я добавил константные ссылки и у меня другая фамилия, я Аммаев, а не Юрченко

**Comment by Sherevv** [comment]
Да, еще +- в базовом классе сделайте методами, а не дружественной функцией
